<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IssueTracker</class>
 <widget class="QWidget" name="IssueTracker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>508</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IssueTracker</string>
  </property>
  <widget class="QListWidget" name="issueList">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>241</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="AddButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>90</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>90</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Report Issue</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RemoveButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>140</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>140</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove Issue</string>
   </property>
  </widget>
  <widget class="QPushButton" name="resolveButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>360</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Resolve</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="descriptionBox">
   <property name="geometry">
    <rect>
     <x>312</x>
     <y>19</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
